dbms/src/DataStreams/HashJoinProbeBlockInputStream.cpp

```diff
@@ -19,35 +19,42 @@ namespace DB
 {
 HashJoinProbeBlockInputStream::HashJoinProbeBlockInputStream(
     const BlockInputStreamPtr & input,
-    const ExpressionActionsPtr & join_probe_actions_,
+    const JoinPtr & join_,
     const String & req_id)
     : log(Logger::get(req_id))
-    , join_probe_actions(join_probe_actions_)
+    , join(join_)
 {
     children.push_back(input);
```

Comment:
Add runtime check here to make sure `join` is not nullptr?

Reply:
> Add runtime check here to make sure `join` is not nullptr?

done.